### PR TITLE
WoE: implement Kretchee

### PR DESCRIPTION
**Pull request by strib**
Added new effects for capturing and exalting one extra amber.

Seems like this might be a good one to have an extra log message for, but I haven't figured out how to do that yet.  Any tips would be appreciated!

### CODE REVIEW

File: server/game/effects.js [hunk 1]
```diff
@@ -122,6 +122,8 @@ const Effects = {
     restrictHouseChoice: (house) => EffectBuilder.player.static('restrictHouseChoice', house),
     stealFromPool: () => EffectBuilder.player.static('stealFromPool'),
     captureFromPool: () => EffectBuilder.player.static('captureFromPool'),
+    captureOneMoreFromPool: () => EffectBuilder.player.static('captureOneMoreFromPool'),
```
Comment (romanogit) [inline server/game/effects.js:125]: I believe the implementation is good, but we could make this more flexible by adding an amount. You could name, for example: "captureMoreFromPool" or "captureExtraFromPool"

File: server/game/GameActions/ExaltAction.js [hunk 2]
```diff
@@ -16,10 +16,18 @@ class ExaltAction extends AddTokenAction {
...
 
     getEvent(card, context) {
+        let player = this.player || context.player.opponent;
         return super.createEvent(
             'onExalt',
             { card: card, context: context, amount: this.amount },
-            () => card.addToken('amber', this.amount)
+            () => {
+                let extra = 0;
+                if (this.amount > 0 && player.anyEffect('exaltOneMoreFromPool')) {
```
Comment (romanogit) [inline server/game/GameActions/ExaltAction.js:25]: You need to count the effects because there could be two or more Kretchee in play

**Review by romanogit — CHANGES_REQUESTED** [review]

File: server/game/GameActions/ExaltAction.js (hunk 2 above)
Comment (strib) [reply server/game/GameActions/ExaltAction.js]: Oops, great catch.  Fixed.

File: server/game/effects.js (hunk 1 above)
Comment (strib) [reply server/game/effects.js]: Good suggestion, done!

**Pull request merged**